### ISSUE TITLE
FEATURE REQUESTS: Line position with bookmark always nearest to top

### ISSUE THREAD

**Issue by AmelianceSkyMusic** [koalamer/vsc-labeled-bookmarks#6]
Line position with bookmark always nearest to top of the screen after click on bookmark in bookmarks panel.
Not under the edge, but 1-2 lines below

**Comment by koalamer** [comment]
Yeah, the basic navigation is unfortunately not smart enough to do this on its own, but I could fake it.

It  will probably be a setting to keep the navigation target "near the top" (your suggestion), "in the middle" and "default" (the current one). I don't know if "near the bottom" makes sense, but maybe that too. (Vim zt, zz, zb)

**Comment by abulka** [comment]
Would also be nice to have the focus shift to the source code (whenever you click on a bookmark), so that your eye is drawn to the position of the blinking cursor.

**Comment by mrlee14** [comment]
yes please...
